src/answers/services/answers.service.ts

```diff
@@ -158,7 +158,7 @@ export class AnswersService {
       areaTeams.includes(team.id.toString()),
     );
     // extract all user ids
-    const userIds = [];
+    const userIds: (mongoose.Types.ObjectId | string | undefined)[] = [];
```

Comment:
```suggestion
    const userIds: (mongoose.Types.ObjectId)[] = [];
```

Reply:
And change line 174 to else userIds.push(new mongoose.Types.ObjectId(loggedUser.id);